src/Collection/CommitteeMembershipCollection.php

```diff
@@ -77,6 +77,13 @@ public function getCommitteeSupervisorMemberships(): self
         });
     }
 
+    public function getCommitteeUnapprovedMemberships(): self
```

Comment:
What do you think to rename the `getCommitteeUnapprovedMemberships`method to `getCommitteeMembershipsInPending` ?

Reply:
@JeremieSamson Are you sure that we need a list of only pending committees without pre-approved and pre-refused ones?

Reply:
is it confirmed ? @JeremieSamson